numpy/core/src/multiarray/multiarraymodule.c

```diff
@@ -4235,35 +4235,13 @@ setup_scalartypes(PyObject *NPY_UNUSED(dict))
         return -1;                                                      \
     }
 
-/*
- * In Py3K, int is no longer a fixed-width integer type, so don't
- * inherit numpy.int_ from it.
- */
-#if defined(NPY_PY3K)
-#define INHERIT_INT(child, parent2)                                     \
-    SINGLE_INHERIT(child, parent2);
```

Comment:
s/INHERIT_INT/SINGLE_INHERIT/ below leads to 

if SIZE = OTHER_SIZE:
  SINGLE_INHERIT(a, b)
else:
  SINGLE_INHERIT(a, b)

which can be simplified